css/blocks/popup-order/checkbox-info.less

```
@@ -0,0 +1,22 @@
+// Уведомления checkbox
+
+input[type=checkbox] {display: none;}
+
+input[type=checkbox] + label:before {
```

чекбок и все другие элементы форм - это часто используемые блоки на сайте. Поэтому лучше их сразу выделить в отдельные независимые блоки. Положить в /common и назвать, не привязываясь к месту их использования.
Например,  `/common/checkbox`, `/common/radio`, `/common/select` и т.п.

---

```suggestion
.checkbox__input + .checkbox_label:before {
```
То есть не надо привязывать стили к тегам, лучше к классам. Иначе, когда тебе понадобиться где-нибудь использовать чекбокс с совершенно другим дизайном, то придется переопределять все эти CSS-свойства